tasks/google/add-arch-linux/task.yaml

```diff
@@ -0,0 +1,27 @@
+summary: Add new arch linux image to google account
+
+systems: [ ubuntu-* ]
+
+environment:
+    IMAGE: arch-linux-64-v20180110
+    FAMILY: arch-linux-64
+    DESCRIPTION: Arch Linux 64 bits
+    IMAGE_FILE: arch-v4.14.12
+
+execute: |
+    . "$TESTSLIB/google.sh"
+
+    if ! [[ "$SPREAD_SYSTEM" == ubuntu-* ]]; then
+        echo "System used to create latest arch linux has to be ubuntu"
+        exit 1
+    fi
+
+    # Download arch linux image from arch_linux bucket
+    gsutil cp "gs://arch_linux/${IMAGE_FILE}.tar.gz" "${IMAGE_FILE}.tar.gz"
```

Comment:
If I understand that correctly, we are not really building the image from scract, but rather pulling one that's already in our store and repacking it?

I hope that we'd be able to update the image regularly at some point.

Reply:
Yes, I am using a prebuilt image as a base. 
Then by using the update-arch-linux script we will be able to update the image regularly. We will have a task in spread-cron for this when spread-cron is fixed.